xdsl/dialects/llvm.py

```diff
@@ -133,15 +132,26 @@ class IntToPtrOp(Operation):
 
     output: Annotated[OpResult, LLVMPointerType]
 
-    @classmethod
-    def get(cls,
-            input: SSAValue | Operation,
-            ptr_type: Attribute | None = None):
+    @staticmethod
+    def get(input: SSAValue | Operation, ptr_type: Attribute | None = None):
         if ptr_type is None:
             ptr_type = LLVMPointerType.opaque()
         else:
             ptr_type = LLVMPointerType.typed(ptr_type)
-        return cls.build(operands=[input], result_types=[ptr_type])
+        return IntToPtrOp.build(operands=[input], result_types=[ptr_type])
+
+
+@irdl_op_definition
+class PtrToIntOp(Operation):
+    name = "llvm.ptrtoint"
+
+    input: Annotated[Operand, LLVMPointerType]
+
+    output: Annotated[OpResult, IntegerType]
+
+    @staticmethod
+    def get(arg: SSAValue | Operation, int_type: Attribute = i64):
```

Comment:
```suggestion
    def get(arg: SSAValue | Operation, int_type: Attribute = i64) -> PtrToIntOp:
```

Reply:
Not sure on these, makes copy-pasting more difficult (don't know if that's a bad thing)

Reply:
Copy-pasting of what? :sweat_smile: 
Of the code itself?

Reply:
Well, if you add a new `get` that's quite similar to an older one, you'd have to replace one more occurance of `OldType` with `NewType`^^